misc/SW_WA_2021_2024/guilderton_to_geraldton/swals/README.md

```diff
@@ -0,0 +1,390 @@
+# See ../analysis/check_log_files for more info.
+#
+# Make rasters in the usual way and check the results, e.g. on a node with 48 cores from inside the multidomain_dir
+Rscript ../../../post_process/make_rasters.R '.' 48 max_stage max_flux max_speed time_of_max_stage
+gdalbuildvrt -resolution highest max_stage_all.vrt max_stage_domain*.tif
+gdalbuildvrt -resolution highest max_speed_all.vrt  max_speed_domain_*.tif
+gdalbuildvrt -resolution highest time_of_max_stage_all.vrt  time_of_max_stage_domain_*.tif
+gdalbuildvrt -resolution highest max_flux_all.vrt  max_flux_domain_*.tif
+# If you saved temporal parameters you could additionally create the last timestep rasters
+gdalbuildvrt -resolution highest last_timestep_UH_all.vrt last_time_UH_domain_*.tif
+gdalbuildvrt -resolution highest last_timestep_VH_all.vrt last_time_VH_domain_*.tif
+#
+# Process gauges for the historical validation cases (two steps for each event, figures end up in the ./plots/ folder).
+# Example for Sumatra 2004
+cd plots
+Rscript process_gauges_sumatra2004.R ../OUTPUTS/path_to_multidomain_dir_sumatra2004
+cd ..
+# plot with varying y-scale
+Rscript plot_gauges_perth_sumatra2004.R OUTPUTS/path_to_multidomain_dir_sumatra2004/gauge_RDS_file_name.RDS
+# uniform y range of 1m
+Rscript plot_gauges_perth_sumatra2004.R OUTPUTS/path_to_multidomain_dir_sumatra2004/gauge_RDS_file_name.RDS 1.0
+
+# As above, Sumatra 2005
+cd plots
+Rscript plots/process_gauges_sumatra2005.R ../OUTPUTS/path_to_multidomain_dir_sumatra2005
+cd ..
+# varying y range
+Rscript plot_gauges_perth_sumatra2005.R OUTPUTS/path_to_multidomain_dir_sumatra2005/gauge_RDS_file_name.RDS
+# uniform y range of 0.3m
+Rscript plot_gauges_perth_sumatra2005.R OUTPUTS/path_to_multidomain_dir_sumatra2005/gauge_RDS_file_name.RDS 0.3
+#
+
+
+#
+# Run the random scenarios
+#
+
+# Set-up a script that can create all the PBS submission scripts, for all
+# scenarios you want to run. 
+vim create_random_ptha_qsub_scripts_sealevel60cm.R
+# Then make the scripts.
+Rscript create_random_ptha_qsub_scripts_sealevel60cm.R
+# Eyeball a few scripts. 
+# If they look OK, then submit one to see if it works.
+qsub run/midwest_sealevel60cm_1001.pbs
+# It's probably best to move the submitted scripts to another directory, to keep
+# track of which have been submitted.
+mkdir run/midwest_submitted_jobs
+mv run/midwest_sealevel60cm_1001.pbs run/midwest_submitted_jobs/
+
+# Once you're convinced that it's running correctly (including tarring the
+# output folders) you can submit all jobs with e.g.
+for i in run/midwest_sealevel60cm_10*.pbs; do echo $i; qsub $i; mv $i run/midwest_submitted_jobs/; done
+# Be careful as this typically uses lots of compute.
+# * Avoid overwhelming the queue, and ensure you have enough disk/inodes/KSU.
+# * In practice you might want to submit in batches of 10 (or whatever) to deal
+#   with those limits, and double-check everything is OK without using all your
+#   compute quota.
+
+
```

Comment:
Somewhere here, do you need to flag that some runs failed, and there are scripts that treat those cases (at XX location) and here's how you do it.